io.github.openhv.OpenHV.yaml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.openhv.OpenHV
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: openhv
+rename-desktop-file: openhv.desktop
+rename-appdata-file: openhv.metainfo.xml
+rename-icon: openhv
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - shared-modules/lua5.1/lua-5.1.5.json
+  - name: OpenHV
+    sources:
```

Comment:
The preferred practice is to have `sources` defined below everything else.

Reply:
also since we talk order, I would have prefered the openHV to be on top. Because that's the main. For moment I got confused that you had misnamed the app-id because openRA was the first git source.